ethexe/processor/src/host/mod.rs

```diff
@@ -54,8 +54,6 @@ pub(crate) struct InstanceCreator {
 
 impl InstanceCreator {
     pub fn new(runtime: Vec<u8>) -> Result<Self> {
-        gear_runtime_interface::sandbox_init();
```

Comment:
why dont we need to init sandbox anymore?

Reply:
The only reason of the function existence is necessity to initialize `wasmer` trap handler before `lazy-pages` one. Now it happens right in `lazy-pages` crate